examples/equilibrium.py

```diff
@@ -84,15 +88,17 @@
 
 # +
 resids = [np.array(e).ravel() for e in eqs]
-from matplotlib import pyplot as plt
+
 Na = len(dist)
-for i,eq in enumerate(equils):
-    plt.plot(kvec, kvec-eq, label=f"β={dist[i][1]['β']:.3f}")
-plt.plot(kvec, kvec-sum(resids,0)/Na, linestyle='--', color='black', label='Total Demand')
-plt.plot(kvec, kvec, color='black', label='Total Supply')
-plt.legend(loc='upper right')
+# CDC 20191027: Not sure what this is supposed to do, but it has some bugs, so commented out
```

Comment:
That piece of code was supposed to plot total aggregate supply for capital as a function of aggregate capital, as well as the supply of capital as if all agents where all of type i, for the various types i